### ISSUE TITLE
Unable to use local apollo cache before a user has signed in

### ISSUE THREAD

**Issue by mllrmat** [aws-amplify/amplify-js#1793]
I am trying to use the local apollo cache. This works fine as soon as a user has signed in via cognito. But before sign in a "Possible Unhandled Promise Rejection (id: 0): No current user" error is thrown. 

This is undesirable as it makes it impossible to use the local apollo cache as a local state management mechanism as advertised by Apollo: https://www.apollographql.com/docs/react/essentials/local-state.html

The query: 
```
const getUserUuid = gql`
  {
    user_uuid @client
  }
`;
```
I am trying to get the value via Apollo Query Component, eg.: 
```
<Query query={getUserUuid} >
{({ data, client }) => (
    <Text>{data.user_uuid}</Text>
)}
</Query>
```
**Expected behavior**
Getting the value of user_uuid from the local cache without exception. The local cache should be completely independent of the login status of the client application.

**Comment by elorzafe** [comment]
Hi @mllrmat how are you integrating Amplify Auth with Apollo cache?

**Comment by jordanranz** [comment]
Closing due to inactivity. Please open a new issue if you are still blocked on this issue.
